tests/validate/test_filter_and_accumulate_validation_errors.py

```diff
@@ -0,0 +1,41 @@
+from cerberus import Validator
+from chispa import assert_df_equality
+from pyspark.sql import Row
+
+from cishouseholds.pyspark_utils import ListAccumulator
+from cishouseholds.validate import filter_and_accumulate_validation_errors
+
+
+def test_filter(spark_session):
+    schema = "id string, value integer"
+    input_df = spark_session.createDataFrame([("id1", 1), ("idtwo", 1), ("id3", -1)], schema)
+
+    expected_df = spark_session.createDataFrame([("id1", 1)], schema)
+
+    validator = Validator({"id": {"type": "string", "regex": r"id\d"}, "value": {"type": "integer", "min": 0}})
```

Comment:
Should this be `PySparkValidator` or when should that class be used? Should PySparkValidator be used for consistency - or is only needed when a `datetime` type is used.

Reply:
Yep, it is only needed when `datetime` type is used. But agree with your comment above that this should be tested... 😢